code/src/App.js

```diff
@@ -1,9 +1,23 @@
-import React from 'react'
+import React, { useState } from "react";
+import QuestionAboutName from "./components/QuestionAboutName";
+import QuestionAboutPainting from "./components/QuestionAboutPainting";
+import QuestionAboutAuthor from "./components/QuestionAboutAuthor";
+import Summary from "./components/Summary";
 
 export const App = () => {
+  const [answers, setAnswers] = useState([]);
+  const [count, setCount] = useState(0);
+  const onFormSubmit = (value) => {
+    answers.push(value);
```

Comment:
I think the answers array is used to hold both answers and results of your validation which is a bit confusing to me, but it's also smart thinking! Good job :D